gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Oct 13 19:11:10 CEST 2016
+#Sat Oct 21 14:56:39 AEDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```

Comment:
This alone does not update `gradle/wrapper/gradle-wrapper.jar` and `gradlew` files

You'll need to run Gradle, ie `./gradlew --version`